pysrc/bytewax/dynamodb_output.py

```diff
@@ -0,0 +1,43 @@
+import boto3
+
+from bytewax.outputs import ManualOutputConfig
+
+
+def output_builder(table):
+    """Construct an output function to map Bytewax's (epoch, item) structure
+    to boto3's required structure"""
+    dynamodb = boto3.resource("dynamodb")
+    table = dynamodb.Table(table)
+
+    def put_item(item):
+        _epoch, item = item
+        table.put_item(Item=item)
```

Comment:
Do you think it's worthwhile to make this

```py
_epoch, kwargs = epoch_kwargs
table.put_item(**kwargs)
```

Then this allows users of this output config to have access to the fancier stuff of `put_item` like `ConditionExpression`.